sample.py

```diff
@@ -38,6 +40,11 @@
 geocode = l.geocode(['New York', 'Singapore'])
 pprint(geocode)
 
+# Loklak Map API
+png_data = l.map(17.582729, 79.118320, 512, 512, 12, 'Hello')
+path = tempfile.mkstemp(suffix=".png")[1]
+open(path, 'wb').write(png_data)
```

Comment:
Creating a file isn't the required thing from this. what's needed is to return the file content, the raw content.